python/README.md

```diff
@@ -6,6 +6,7 @@
 * [BigQueryToGCS](/python/dataproc_templates/bigquery/README.md)
 * [HiveToBigQuery](/python/dataproc_templates/hive/README.md)
 * [HiveToGCS](/python/dataproc_templates/hive/README.md)
+* [TextToBigQuery](/python/dataproc_templates/gcs/README.md)
```

Comment:
Can we add similar in dataproc-templates/README.md?

Reply:
Done.